src/olympia/abuse/cinder.py

```diff
@@ -186,7 +184,7 @@ def _send_create_decision(
             ),
         }
         response = requests.post(url, json=data, headers=self.get_cinder_http_headers())
-        if response.status_code == success_code:
+        if response.status_code == 201:
```

Comment:
Since we already had to change this value due to cinder changing their response code, we should definitely use `response.raise_for_status()` instead of checking for a specific exact code. We only care that it is a successful code and that we get the json. Who care if the number ends in 0 1 or 4.